unimus-backup-exporter.sh

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
+
+#This is a Unimus to Git API to export your backups to your Git Repo
+
+
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
+backup_dir=$script_dir/backups
+
+
+
+#Imports from env
+set -a # automatically export all variables
+source unimus-backup-exporter.env
+set +a
+
+
+#HashTable for all devices
+declare -A devices
+log=$script_dir/unimus-backup-exporter.log
+
+
+function echoGreen() { echo -e "${green}$1${reset}"; }
```

Comment:
consistent formatting across script

Reply:
Formatting has been made more consistant. 

Also not listed in this PR, echoGreen, echoYellow, and echoRed now display proper colors.